system/HTTP/Files/UploadedFile.php

```diff
@@ -141,20 +141,25 @@ public function move(string $targetPath, ?string $name = null, bool $overwrite =
         $destination = $overwrite ? $targetPath . $name : $this->getDestination($targetPath . $name);
 
         try {
-            move_uploaded_file($this->path, $destination);
+            $this->hasMoved = move_uploaded_file($this->path, $destination);
         } catch (Exception $e) {
```

Comment:
How does `move_uploaded_file()` throws an Exception? From the PHP docs, the function throws a warning when a valid file cannot be moved for some reason.

Reply:
Warnings turn into Exceptions by the framework.
https://www.php.net/manual/en/function.set-error-handler.php

Reply:
Yeah, I missed that. 👍